### PR TITLE
Added individual answers tab for each survey

### PR DESCRIPTION
**Pull request by joharkhan99**
Hey @Ryczko Please check this PR for the Individual Results tab, allowing users to view each survey response individually. It includes navigation controls with "Next" and "Previous" buttons, as well as an input field to jump to specific responses. Each question displays the corresponding answer or "No response" if unanswered, with a message for cases where no responses are available. Navigation and display have been tested to ensure accurate handling of responses.

[Demo Video 1](https://nimb.ws/TgIHY7V)
[Demo Video 2](https://nimb.ws/nhV7BcS)

### CODE REVIEW

File: src/features/surveys/features/SurveyResults/components/IndividualResults/IndividualResults.tsx [hunk 1]
```diff
@@ -1,5 +1,152 @@
...
+                    )}
+                    onMouseEnter={() => {
+                      console.log('OK');
+                    }}
+                    onMouseLeave={() => {
+                      console.log('OK');
+                    }}
+                    onClick={() => {
+                      console.log('OK');
+                    }}
```
Comment (Ryczko) [inline src/features/surveys/features/SurveyResults/components/IndividualResults/IndividualResults.tsx:111]: Please make all callbacks for each question type optional, and just don't pass it 😊

**Review by Ryczko — COMMENTED** [review]
Hey, looks good 🙂. Please provide that small changes

File: src/features/surveys/features/SurveyResults/components/IndividualResults/IndividualResults.tsx (hunk 1 above)
Comment (joharkhan99) [reply src/features/surveys/features/SurveyResults/components/IndividualResults/IndividualResults.tsx:111]: Thanks for looking at my request. Sure, I will make all question type callbacks optional and pass only when needed. Let me know if there's anything else!

**Comment by joharkhan99** [comment]
@Ryczko, I've made the changes based on your feedback. When you have a moment, could you please take another look? If there's anything further to improve, I'd be happy to make adjustments. Thank you!

**Pull request merged**